tests/unit/test_utils.py

```diff
@@ -771,22 +783,6 @@ def test_no_remote(cls, local_git_repo: Path, root_logger):
         assert not val
         assert "Failed to get remote branch" in msg
 
-    @classmethod
-    def test_untracked_fmf_root(cls, local_git_repo: Path, root_logger):
```

Comment:
Why is this this test removed? People can reset git index. Or use 'fmf init' or create fmf root manually.

`validate_git_status`  is still responsible to detect that fmf root is not part of the git repo.

Reply:
Agreed, it would be better to extend the test with `git reset .fmf` instead.

Reply:
I misunderstood the purpose of this test and thought it can be safely deleted after my change. Thank you for the clarification. I returned the test with a small adjustment so that fmf root is not tracked.